src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/__tests__/ContactDiaryStoreTests.swift

```diff
@@ -632,7 +632,7 @@ class ContactDiaryStoreTests: XCTestCase {
 		}
 
 		let expectedString = """
-			Kontakte der letzten 14 Tage (02.12.2020 - 15.12.2020)
+			Kontakte der letzten 15 Tage (01.12.2020 - 15.12.2020)
```

Comment:
This will fail in multi-language tests. Please localize OR (simple solution) skip this test if `locale != "de_DE"`.